src/common/dpi_aware.cpp

```diff
@@ -5,6 +5,11 @@
 
 namespace DPIAware
 {
+    HRESULT GetScreenDPI(HMONITOR monitor, UINT& dpi_x, UINT& dpi_y)
```

Comment:
Why not just calling the `GetDpiForMonitor`? :)

Reply:
As a matter of fact this is no longer needed, but initially I've named it with `screen` to align with `GetScreenDPIForPoint` and `GetScreenDPIForWindow`. I'll remove this.

Reply:
Removed.